code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -75,11 +76,10 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	GLOB.parasites -= src
 	return ..()
 
-/mob/living/simple_animal/hostile/guardian/proc/setthemename(pickedtheme) //set the guardian's theme to something cool!
-	if(!pickedtheme)
-		pickedtheme = pick("magic", "tech", "carp")
+/mob/living/simple_animal/hostile/guardian/proc/setthemename() //set the guardian's theme to something cool!
```

Comment:
This is still called with an argument in two places, liable to break stuff if those arguments are now being ignored.